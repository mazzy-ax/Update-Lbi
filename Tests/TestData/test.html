<!-- html fragment -->
    more <!-- #BeginLibraryItem "/Library/testdiv.lbi" -->is it indented?<!-- #EndLibraryItem --><!-- #BeginLibraryItem "/Library/testdiv.lbi" -->is it indented?<!-- #EndLibraryItem -->
