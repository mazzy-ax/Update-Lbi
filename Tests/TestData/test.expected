<!-- html fragment -->
    more <!-- #BeginLibraryItem "/Library/testdiv.lbi" -->
    <div>
        div content
    </div>
    <!-- #EndLibraryItem --><!-- #BeginLibraryItem "/Library/testdiv.lbi" -->
    <div>
        div content
    </div>
    <!-- #EndLibraryItem -->
